CHANGELOG.yaml

```diff
@@ -1,3 +1,7 @@
+master:
+    fixed bugs:
+     - GH-2509 Fix to not store memory heavy objects in summary instead give options
```

Comment:
I feel this is more of an improvement, than a bug fix. We're adding a new flag! We should add it under `new features:`.
What do you think?